app/templates/ember-community-survey-2021.hbs

```diff
@@ -11,12 +11,12 @@
   <div class="mt-3">
     <a
       class="es-button"
-      href="#"
+      href="https://tilde.wufoo.com/forms/2021-emberjs-community-survey"
```

Comment:
To confirm, line 17 shows that we're using `disabled` attribute. Should this line be removed now that we have a link?

Reply:
yes, thanks for catching that